pom.xml

```diff
@@ -195,6 +196,17 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>com.google.api</groupId>
+        <artifactId>api-common</artifactId>
+        <version>${project.api-common.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+          </exclusion>
+        </exclusions>
```

Comment:
@diegomarquezp I meant if you can add a comment about the exclusion into the pom.xml and not in the release notes. 

I think us deciding the exclude guava is more of a implementation detail for us and not relevant for users.